asp_attendance/models/daily_reporting.py

```diff
@@ -17,6 +17,9 @@ class DailyReporting(models.Model):
     holiday_id = fields.Many2one("hr.leave")
     is_late_check_in = fields.Boolean(compute="_compute_late_check_in", store=True)
     is_after_10_check_in = fields.Boolean(compute="_compute_after_10_check_in", store=True)
+    hours_in_schedule = fields.Float()
```

Comment:
```suggestion
    hours_to_work = fields.Float(string="Hours To Work")
```
This name may be better to describe what it stores.